sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -384,6 +387,8 @@ public partial class BlobCopyFromUriOptions
         public BlobCopyFromUriOptions() { }
         public Azure.Storage.Blobs.Models.AccessTier? AccessTier { get { throw null; } set { } }
         public Azure.Storage.Blobs.Models.BlobRequestConditions DestinationConditions { get { throw null; } set { } }
+        public Azure.Storage.Blobs.Models.BlobImmutabilityPolicy ImmutabilityPolicy { get { throw null; } set { } }
+        public bool LegalHold { get { throw null; } set { } }
```

Comment:
if these apply to destination then should we prefix them with `Destination`?
Btw. we used `ShouldSealDestination` terminology before, should we use something like that for `LegalHold`?

Reply:
Changed to `DestinationImmutabilityPolicy`.